package.json

```diff
@@ -24,9 +176,8 @@
   "dependencies": {
     "ocaml": "~4.7.0",
     "@esy-ocaml/reason": "3.4.0",
-    "isolinear": "*",
```

Comment:
A few lines down we have an explicit version specified so I removed this duplicate. (If it should be the specific one that is removed and this one kept, just shout up).